notebooks/vertex_genai/solutions/grounding_vertex_agent_builder.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<ul><li> Is it feasible to add a programatic version to these UI instructions? I think both are valuable. </li><li> In the instructions, I'd suggest concrete values to input: e.g. "enter the name of your app" --&gt; "enter the name of your app (for example: Google Store App)</li><li>Maybe specify what "advanced website indexing" is and why we need to turn it off. Same thing for "Advanced LLM features"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>